### DISCUSSION TITLE
how 2 use?

### DISCUSSION THREAD

**Discussion by joe-bro-1** [pixijs/pixijs#9003] [Q&A]
how do i use *pixi.js* with *node.js*

**Comment by domis86** [comment]
If you mean to run PixiJS not in browser but in node.js then:

https://github.com/pixijs/pixijs/tree/dev/bundles/pixi.js-node

**Reply by joe-bro-1** [reply]
i want 2 use in browser but have a *node.js* server instead of a *cdn*

**Reply by miltoncandelero** [reply]
So what you are looking for is how to make a static serving fileserver with node.js (basically, how to make your own content delivery server)
nothing to do with pixi 😅

**Reply by joe-bro-1** [reply]
no i want 2 know how 2 use pixi
side-topic i'm  not the admin in my computer and the admin has a firewall set so I can’t access a cdn so I want 2 use the pixi *node.js* library
